proto/interchain_security/ccv/provider/v1/provider.proto

```diff
@@ -106,6 +106,12 @@ message ConsumerAdditionProposal {
   repeated string allowlist = 18;
   // Corresponds to a list of provider consensus addresses of validators that CANNOT validate the consumer chain.
   repeated string denylist = 19;
+  // Corresponds to the minimal amount of (provider chain) stake required to validate on the consumer chain.
+  uint64 min_stake = 20;
+  // Corresponds to the maximal rank in the provider chain validator set that a validator can have to validate on the consumer chain.
+  uint32 max_rank = 21;
```

Comment:
Example: 
* Set it to 200 if you want only active validators + the first 20 validators with the most stake to validate
* Set it to 100 if you only want the 100 active validators with the most stake to be able to validate
is it confusing?

Reply:
With the example is easier to understand. For me a higher rank sounds like a positive which means that the top validator has highest rank. I know that this might not be true. So to address the potential confusion of users, either change the name to something else or just add the examples to the docstring in the proto file and to the documentation of power shaping.

Reply:
Added